scripts/apps/authoring/authoring/directives/ArticleEditDirective.js

```diff
@@ -280,17 +280,25 @@ export function ArticleEditDirective(
                  *
                  * @description Opens the Change Image Controller to modify the image metadata.
                  */
-                scope.editMedia = (defaultTab = 'view', hideTabs = []) => {
+                scope.editMedia = (defaultTab = 'view') => {
+                    let hideTabs = [];
+
                     scope.mediaLoading = true;
 
+                    if (scope.item.type === 'video' || scope.item.type === 'audio') {
+                        hideTabs = ['image-edit', 'crop'];
```

Comment:
Use direct checks:
```
if (scope.item.type === 'image') {
    showTabs = ['image-edit', 'crop']
```

When using negative checks, future changes break stuff easily. If I added a new type say 'powerpoint slides' cropping and image-edit would enable automatically which wouldn't be the case if using direct checks.